plugins/catalog-backend-module-incremental-ingestion/README.md

```diff
@@ -42,57 +42,25 @@ The Incremental Entity Provider backend is designed for data sources that provid
 ## Installation
 
 1. Install `@backstage/plugin-catalog-backend-module-incremental-ingestion` with `yarn --cwd packages/backend add @backstage/plugin-catalog-backend-module-incremental-ingestion` from the Backstage root directory.
-2. In your catalog.ts, import `IncrementalCatalogBuilder` from `@backstage/plugin-catalog-backend-module-incremental-ingestion` and instantiate it with `await IncrementalCatalogBuilder.create(env, builder)`. You have to pass `builder` into `IncrementalCatalogBuilder.create` function because `IncrementalCatalogBuilder` will convert an `IncrementalEntityProvider` into an `EntityProvider` and call `builder.addEntityProvider`.
 
-```ts
-const builder = CatalogBuilder.create(env);
-// incremental builder receives builder because it'll register
-// incremental entity providers with the builder
-const incrementalBuilder = await IncrementalCatalogBuilder.create(env, builder);
-```
+2. Add the following code to the `packages/backend/src/index.ts` file:
 
-3. After building the regular `CatalogBuilder`, build the incremental builder:
+```diff
+++import { catalogModuleCustomIncrementalIngestionProvider } from './extensions/catalogCustomIncrementalIngestion';
```

Comment:
There are a few places with double plus signs, one is enough.

```suggestion
+ import { catalogModuleCustomIncrementalIngestionProvider } from './extensions/catalogCustomIncrementalIngestion';
```

Reply:
Thanks @jhaals. Double plus signs removed.